### PR TITLE
Expense payee total payout sum

### PR DESCRIPTION
**Pull request by juliekitty**
Resolve https://github.com/opencollective/opencollective/issues/4491
API PR: https://github.com/opencollective/opencollective-api/pull/6774

# Description

3rd step: display the Expense payee total payout sum for the calendar year, splitted in RECEIPT, GRANT and INVOICES

# Screenshots
Capture d’écran 2021-11-04 à 13 08 09

### CODE REVIEW

**Comment by znarf** [comment]
For the lint issue, make sure to rebase, Also rebase on the API and update graphql schema here.

File: pages/expense.js [hunk 1]
```diff
@@ -90,6 +98,18 @@ const expensePageQuery = gqlV2/* GraphQL */ `
             valueInCents
             currency
           }
```
Comment (znarf) [inline pages/expense.js:100]: Can we remove `totalExpensesReceived: totalAmountReceived` now?

File: pages/expense.js [hunk 2]
```diff
@@ -90,6 +98,18 @@ const expensePageQuery = gqlV2/* GraphQL */ `
...
+          totalPaidExpenseInvoices: totalPaidExpense(expenseType: [INVOICE], dateFrom: $totalPaidExpenseDateFrom) {
```
Comment (znarf) [inline pages/expense.js:101]: I would ask the value to be converted in the currency of the Collective or Host paying the expense. `currency: $collectiveCurrency`

File: pages/expense.js [hunk 3]
```diff
@@ -68,7 +70,13 @@ const messages = defineMessages({
 });
 
 const expensePageQuery = gqlV2/* GraphQL */ `
-  query ExpensePage($legacyExpenseId: Int!, $draftKey: String, $offset: Int, $totalExpensesReceivedDateFrom: DateTime) {
+  query ExpensePage(
+    $legacyExpenseId: Int!
+    $draftKey: String
+    $offset: Int
+    $totalExpensesReceivedDateFrom: DateTime
```
Comment (znarf) [inline pages/expense.js:77]: Do we still need this?

Comment (juliekitty) [reply pages/expense.js]: no, i delete it now

**Review by znarf — APPROVED** [review]
Looks good!

**Pull request merged**